homeassistant/helpers/schema_config_entry_flow.py

```diff
@@ -68,6 +68,32 @@ class SchemaFlowMenuStep:
     options: list[str] | dict[str, str]
 
 
+def schema_with_suggested_values(
```

Comment:
I think that having this method available to the public would be useful, instead of keeping it hidden `SchemaCommonFlowHandler`.

Maybe it should be moved to a different helper?
cc @emontnemery

Reply:
Why are we introducing this new helper in this PR? Seems like a superfluous change when we're merely fixing an incorrect option in OpenUV. Why not a separate PR for this new helper?

Reply:
I found the bug when testing the helper...
But you are right - I'll open a new PR just for the helper.